src/io/flutter/logging/FlutterLogColorPage.java

```diff
@@ -0,0 +1,101 @@
+                                          "<finer>02-02 18:52:57.132: FINER/FlutterIsCool: log message</finer>\n" +
+                                          "<fine>02-02 18:52:57.132: FINE/FlutterIsCool: log message</fine>\n" +
+                                          "<config>02-02 18:52:57.132: CONFIG/FlutterIsCool: log message</config>\n" +
+                                          "<info>02-02 18:52:57.132: INFO/FlutterIsCool: log message</info>\n" +
+                                          "<warning>02-02 18:52:57.132: WARNING/FlutterIsCool: log message</warning>\n" +
+                                          "<severe>02-02 18:52:57.132: SEVERE/FlutterIsCool: log message</severe>\n" +
+                                          "<shout>02-02 18:52:57.132: SHOUT/FlutterIsCool: log message</shout>";
+
+  static {
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("none", FlutterLogConstants.NONE_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("finest", FlutterLogConstants.FINEST_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("finer", FlutterLogConstants.FINER_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("fine", FlutterLogConstants.FINE_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("config", FlutterLogConstants.CONFIG_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("info", FlutterLogConstants.INFO_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("warning", FlutterLogConstants.WARNING_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("severe", FlutterLogConstants.SEVERE_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("shout", FlutterLogConstants.SHOUT_OUTPUT_KEY);
+  }
+
+  @NotNull
+  private static final AttributesDescriptor[] ATTRIBUTES_DESCRIPTORS =
+    new AttributesDescriptor[]{
+      new AttributesDescriptor(FlutterBundle.message("none.level.title"), FlutterLogConstants.NONE_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("finest.level.title"), FlutterLogConstants.FINEST_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("finer.level.title"), FlutterLogConstants.FINER_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("fine.level.title"), FlutterLogConstants.FINE_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("config.level.title"), FlutterLogConstants.CONFIG_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("info.level.title"), FlutterLogConstants.INFO_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("warning.level.title"), FlutterLogConstants.WARNING_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("severe.level.title"), FlutterLogConstants.SEVERE_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("shout.level.title"), FlutterLogConstants.SHOUT_OUTPUT_KEY)};
+
+  @Override
+  @NotNull
+  public String getDisplayName() {
+    return "Flutter Log";
+  }
+
+  @Override
+  public Icon getIcon() {
+    return AllIcons.Debugger.Console_log;
```

Comment:
Do we want the Flutter Icon here?

I see logcat, chooses Android.

Any idea where (if anywhere) this icon is shown?

Reply:
I set it to null.
Actually, I can't figure out where it displays :|